src/Commands/core/UpdateDBCommands.php

```diff
@@ -203,25 +199,25 @@ public function updateDoOne($module, $number, $dependency_map, &$context)
                     Database::startLog($function);
                 }
 
-                $this->logger()->notice("Update started: $function");
+                Drush::logger()->notice("Update started: $function");
```

Comment:
This seems bad, but if it gets the tests to green, I guess we can circle back to figuring out why the logger reference is gone or corrupt in the subprocess call later.

This is better than making a separate .inc file anyway.